### PR TITLE
docs: Update redhat install instructions

### PR DESCRIPTION
**Pull request by ankatare**
Commit Message: Reference url issues in getting started doc
Additional Description: NA
Risk Level: LOW
Testing: Unit
Docs Changes: Yes
Release Notes:
Platform Specific Features:
[Optional Runtime guard:]
[Optional Fixes #Issue]
[Optional Deprecated:]
[Optional [API Considerations](https://github.com/envoyproxy/envoy/blob/main/api/review_checklist.md):]

### CODE REVIEW

**Review by phlax — CHANGES_REQUESTED** [review]
thanks @ankatare for correcting these links

i also checked the links for the repos themselves and it appears they have also been merged

could you fix those links too ?

also - im thinking that given the instructions are now the same for rhel and centos can we collapse these two sets of instructions to one

**Comment by ankatare** [comment]
i also checked the links for the repos themselves and it appears they have also been merged, could you fix those links too ?
@phlax Here, which links are you reffering here ?

**Comment by ankatare** [comment]
also - im thinking that given the instructions are now the same for rhel and centos can we collapse these two sets of instructions to one

@phlax agree. i will do it

**Comment by phlax** [comment]
> which links are you reffering here ?

the line like:

```
$ sudo yum-config-manager --add-repo https://getenvoy.io/linux/rhel/tetrate-getenvoy.repo
```

**Comment by ankatare** [comment]
@phlax Ok. Thanks.. i will do it.

Just quick thing, i am thinking to do it in different PR, please suggest ?

**Comment by phlax** [comment]
> Just quick thing, i am thinking to do it in different PR, please suggest ?

i think lets do it all here - im happy to review it and i think all of these issues stem from the same objective - ie updating the redhat install instructions to use the latest getenvoy

**Comment by ankatare** [comment]
@phlax Sure

File: docs/root/start/install.rst [hunk 1]
```diff
@@ -57,38 +57,16 @@ using `Get Envoy <https://www.getenvoy.io/>`__.
    To add the nightly repository instead, replace the word ``stable`` with ``nightly``,
    when adding the ``apt`` repository.
 
-Install Envoy on CentOS Linux
+Install Envoy on RPM-based distros: CentOS/RHEL/Suse/Amazon Linux/Fedora etc
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
```
Comment (phlax) [inline docs/root/start/install.rst:61]: underline needs to match the length of the title - can you fix @ankatare and ill review the generated docs - thanks

Comment (ankatare) [reply docs/root/start/install.rst:61]: Sure. doing it. some issues in my local env so could not test it locally.

Comment (ankatare) [reply docs/root/start/install.rst:61]: it again failed :( . let me work on my local env first ...

Comment (phlax) [reply docs/root/start/install.rst:61]: same problem - underline length needs to match title in rst (it can actually be longer, but please dont do that!)

Comment (ankatare) [reply docs/root/start/install.rst]: I hope it is done now

**Comment by ankatare** [comment]
@phlax please suggest on current build failure.

**Comment by phlax** [comment]
rendered docs are here https://storage.googleapis.com/envoy-pr/16630/docs/index.html

**Review by phlax — APPROVED** [review]
lgtm - thanks @ankatare

**Comment by phlax** [comment]
> @phlax please suggest on current build failure.

unrelated windows flake i think

**Comment by ankatare** [comment]
> rendered docs are here https://storage.googleapis.com/envoy-pr/16630/docs/index.html



> > @phlax please suggest on current build failure.
> 
> unrelated windows flake i think

Ok. Thanks for quick review @phlax

**Comment by ankatare** [comment]
@phlax just thinking, anything more need to be done to get it merged in main stream ?

**Comment by phlax** [comment]
all good @ankatare - thanks - merged

**Comment by ankatare** [comment]
@phlax thanks a lot :)

**Pull request merged**